store/src/main/docker/app.yml

```diff
@@ -23,6 +24,18 @@ services:
             - 8600:8600
         command: consul agent -dev -ui -client 0.0.0.0
 
+    consul-config-loader:
+        image: jhipster/consul-config-loader:v0.2.2
```

Comment:
@ahmadiq plz create a new docker image for stakater; we always have our own docker images now